src/targets/swift/target.ts

```diff
@@ -1,14 +1,14 @@
 import type { Target } from '../index.js';
 
-import { nsurlsession } from './nsurlsession/client.js';
+import { urlsession } from './urlsession/client.js';
```

Comment:
Can you also delete the `nsurlsession` directory? Looks like only its test fixtures were left.

Reply:
Maybe I deleted(rename) `nsurlsession` in Swift directory.

`nsurlsession` in Objective-C(objc) is left.

Reply:
ah my bad, i missed that the other files showed up as being renamed.